### PR TITLE
Ja octavian mar 2022 p18

### PR DESCRIPTION
**Pull request by octavianx**
Misc changes, mainly on sweep reading mode  Level2 upper menu.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-translations/157)

### CODE REVIEW

File: ja/koreader.po [hunk 1]
```diff
@@ -10145,26 +10113,24 @@ msgstr "保存していない変更があります。"
...
 msgid "Find first"
-msgstr "ファイルを検索"
+msgstr "最初に検索"
 
 #: frontend/ui/widget/inputdialog.lua:805
 #: frontend/ui/widget/inputdialog.lua:828
-#, fuzzy
 msgid "Found in line %1."
-msgstr "'%1' に一致するファイルが見つかりません。"
+msgstr "Line '%1' に一致する結果が見つかりました。"
```
Comment (Frenzie) [inline ja/koreader.po:10133]: Line? (Just taking a random peek.)

Comment (octavianx) [reply ja/koreader.po:10133]: Line   could mean  `line`  or   `line number`. 
Not sure about the context at this moment.
At  finally stage I  will run through all the lines  , so  just take a compromised solution to not translate this word 'Line'.

Comment (Frenzie) [reply ja/koreader.po:10133]: Ah, but you removed the fuzzy. ;-)

Comment (octavianx) [reply ja/koreader.po:10133]: yeah,  I found fuzzy items will not be shown in the simulator...   :)  . 
Anyway if the criteria will not permit the removing  `fuzzy` under this circumstance, I will put it back till I found out the context.

Comment (Frenzie) [reply ja/koreader.po:10133]: It should be fine, but in principle untranslated and unfinished (fuzzy) is generally what translators will go looking for.

Comment (octavianx) [reply ja/koreader.po:10133]: Got it~

**Pull request merged**